dispatchers.go

```diff
@@ -372,6 +372,10 @@ func workdir(b *Builder, args []string, attributes map[string]bool, flagArgs []s
 		workdir = filepath.Join(string(os.PathSeparator), current, workdir)
 	}
 
+	if trimmed := strings.TrimSuffix(workdir, string(os.PathSeparator)); trimmed != "" {
```

Comment:
This code is a a little confusing, would

```
if workdir != os.PathSepator {
   workdir= strings.TrimSuffix(workdir, string(os.PathSeparator))
} 
```
Make easier to understand?

Reply:
Sure.